<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseWidget</class>
 <widget class="QDialog" name="BaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(97, 75, 46);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-colo:white;</string>
     </property>
     <property name="text">
      <string>基地列表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="BaseList">
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="TrainUnitButton">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background: transparent;
border-image:url(:/pixmap/button/button1.png);
color:white;
}
QPushButton:pressed{
background: transparent;
border-image:url(:/pixmap/button/button2.png);
color:white;
}
QPushButton:disabled{
background: transparent;
border-image:url(:/pixmap/button/button3.png);
color:grey;
}</string>
           </property>
           <property name="text">
            <string>训练单位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="StudyButton">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background: transparent;
border-image:url(:/pixmap/button/button1.png);
color:white;
}
QPushButton:pressed{
background: transparent;
border-image:url(:/pixmap/button/button2.png);
color:white;
}
QPushButton:disabled{
background: transparent;
border-image:url(:/pixmap/button/button3.png);
color:grey;
}</string>
           </property>
           <property name="text">
            <string>研究科技</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>名字：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Name">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>金币：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Coin">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>coin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>产能：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Capacity">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Capacity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>已有单位列表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="UnitList">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="EndButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background: transparent;
border-image:url(:/pixmap/button/button1.png);
color:white;
}
QPushButton:pressed{
background: transparent;
border-image:url(:/pixmap/button/button2.png);
color:white;
}
QPushButton:disabled{
background: transparent;
border-image:url(:/pixmap/button/button3.png);
color:grey;
}</string>
       </property>
       <property name="text">
        <string>结束回合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background: transparent;
border-image:url(:/pixmap/button/button1.png);
color:white;
}
QPushButton:pressed{
background: transparent;
border-image:url(:/pixmap/button/button2.png);
color:white;
}
QPushButton:disabled{
background: transparent;
border-image:url(:/pixmap/button/button3.png);
color:grey;
}</string>
       </property>
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background: transparent;
border-image:url(:/pixmap/button/button1.png);
color:white;
}
QPushButton:pressed{
background: transparent;
border-image:url(:/pixmap/button/button2.png);
color:white;
}
QPushButton:disabled{
background: transparent;
border-image:url(:/pixmap/button/button3.png);
color:grey;
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
